smart-contracts/tests/Challenge_7.t.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Unlicense
+pragma solidity >=0.6.0 <0.9.0;
+
+import 'forge-std/Test.sol';
+
+import '@smart-contracts/Challenge_7.sol';
+
+contract Challenge7 is Test {
+  // using Strings for uint256;
```

Comment:
Also commented out

Reply:
If I want to run my tests the compiler tries to compile all the files even if I select a specific test file only, i.e.

`forge test --match-contract Challenge7`

Because the challenges are incomplete, the contracts won't compile and then I can't run my test and therefore I had to comment them out - I'd imagine this would be easier for the devs doing the challenges also

If you know a workaround for this, let met know!

Reply:
Uncommented them, will also uncomment all in further PR's so you can review them more easily

Reply:
Wat je bijvoorbeeld kan doen als hij enkel op 1 functie niet compiled is die functie eruit commente. Maar ik zie zo direct niet waar het op faalt, anders kijken we woensdag even samen.

Reply:
Fixed by moving challenges to separate folders and creating separate profiles for each challenge